pkgs/development/compilers/llvm/common/libcxx/default.nix

```diff
@@ -163,37 +160,14 @@ stdenv.mkDerivation (
       patchShebangs utils/cat_files.py
     '';
 
-    patches = lib.optionals (lib.versionOlder release_version "16") (
-      lib.optional (lib.versions.major release_version == "15")
-        # See:
-        #   - https://reviews.llvm.org/D133566
-        #   - https://github.com/NixOS/nixpkgs/issues/214524#issuecomment-1429146432
-        # !!! Drop in LLVM 16+
-        (
-          fetchpatch {
-            url = "https://github.com/llvm/llvm-project/commit/57c7bb3ec89565c68f858d316504668f9d214d59.patch";
-            hash = "sha256-B07vHmSjy5BhhkGSj3e1E0XmMv5/9+mvC/k70Z29VwY=";
-          }
-        )
-      ++ [
-        (substitute {
-          src = ../libcxxabi/wasm.patch;
-          substitutions = [
-            "--replace-fail"
-            "/cmake/"
-            "/llvm/cmake/"
-          ];
-        })
-      ]
```

Comment:
This removal doesn't look right in the commit it's in.

Reply:
You mean, because it leaves the Musl patch when it shouldn’t, and I incorrectly deleted the Musl patch as unreferenced in an earlier commit? Yes, good catch; will fix.

Reply:
(Also, a great example of why I’ve been avoiding rebuilds.)

Reply:
This is correct now (in the LLVM 15 removal commit).